Gemfile

```diff
@@ -1,6 +1,6 @@
 source 'https://rubygems.org' do
   gem 'rake'
-  gem 'cocoapods', '1.6.1'
```

Comment:
`~> 1` doesn't really do anything (might as well remove the version constraint completely) because it means "any version >=1".

The `~>` operator means "you're only allowed to pick a version where the **last** digit is bigger". So e.g. `~> 1.6` would mean "1.6 or 1.7 or anything 1.x where x>=6; but not 2.x". `~> 1.2.3` would mean "any 1.2.x where x>=3, but not 1.3 nor newer". So here `~> 1` just means "any x where x>1", which is basically… any version of cocoapods (now that it has been out of its 0.x phase for ages)

Reply:
I locked this to `1.9 or greater` for now – thanks for the reminder 😁